scripts/test

```diff
@@ -1,23 +1,63 @@
 #!/bin/bash
-echo 'Waiting for ES to start'
-until curl --output /dev/null --silent --head --fail http://es:9200; do
-    printf '.'
-    sleep 5
-done
-echo "ES has started"
-
-exit_status=0
-
-echo  "Test max file descriptors"
-expected_max=1048576
-max_file_descriptors=$(curl http://es:9200/_nodes/stats/process?filter_path=**.max_file_descriptors | jq '.. | .max_file_descriptors? | select(type != "null")')
-
-if [ "$max_file_descriptors" -eq $expected_max ]
-then
-  echo "Max file descriptors test passed. Expected $expected_max and got $max_file_descriptors"
-else
-  exit_status=1
-  echo "Max file descriptors test failed. Expected $expected_max but got $max_file_descriptors"
-fi
+get_es_stats() {
+  echo 'Waiting for ES to start'
+  until curl --output /dev/null --silent --head --fail http://es:9200; do
+      printf '.'
+      sleep 5
+  done
+  echo "ES has started"
+
+  echo "Waiting for up to 60 seconds for the cluster status be at least yellow"
+  curl --silent --output /dev/null "http://es:9200/_cluster/health?wait_for_status=yellow&timeout=60s"
```

Comment:
If/when this starts to happen improvements to the debug out can be looked at with that specific purpose in mind. For the moment I'm inclined to leave it.